build/prometheus.yaml

```diff
@@ -68,7 +68,7 @@ serverFiles:
         tls_config:
           ca_file: /var/run/secrets/kubernetes.io/serviceaccount/ca.crt
           # If your node certificates are self-signed or use a different CA to the
-          # master CA, then disable certificate verification below. Note that
+          # primary CA, then disable certificate verification below. Note that
```

Comment:
I think this should be "control plane CA" or "Kubernetes cluster CA".

Reply:
Went with "Kubernetes control plane" - since I'm using "control plane" in a few other places.